### DISCUSSION TITLE
C++ Code Syntax Highlighting Issue in Repo

### DISCUSSION THREAD

**Discussion by celuk** [community/community#28290] [Other Feature Feedback, Questions, & Ideas]
Hi all,
It is weird that C++ code is not highlighted in the following code:

https://github.com/kasirgalabs/ERIC/blob/main/kasirga-compiler-and-elf2encryptedhex/src/elf2encryptedhex/elf2encryptedhex.cpp

The code's file extension is .cpp and there is no syntax error actually. And also, other cpp codes in the repo is highlighted but this just not. Is it because there are 4986 lines? I have searched for similar issues but I couldn't find a issue like my. 
Any help will be appreciated, thanks.

Seyyid

**Comment by jfullstackdev** [comment]
I tried pasting it, in editing mode there is the color highlighting

![image](https://user-images.githubusercontent.com/47092464/182837227-6c3ad065-914b-4546-8fae-76555e8c7eca.png)

so, yes maybe because it's really a big file
